scale-tidb-using-tiup.md

```diff
@@ -147,8 +147,8 @@ If you want to add a TiFlash node to the `10.0.1.4` host, take the following ste
 >
 > When adding a TiFlash node to an existing TiDB cluster, you need to note the following things:
 >
-> 1. Confirm that the current TiDB version supports using TiFlash, otherwise upgrade your TiDB cluster to v5.0.0-rc or higher.
-> 2. Download [pd-ctl](https://download.pingcap.org/tidb-v5.0.0-rc-linux-amd64.tar.gz) and execute the `config set enable-placement-rules true` command to enable the PD's Placement Rules.
+> 1. Confirm that the current TiDB version supports using TiFlash, otherwise upgrade your TiDB cluster to v5.0.0-rc or later versions.
```

Comment:
v5.0.0-rc is not GA yet. We generally recommend users to use the stable version.
```suggestion
> 1. Confirm that the current TiDB version supports using TiFlash. Otherwise, upgrade your TiDB cluster to v4.0 or later versions.
```